### PR TITLE
replace hub ready doc with checklist and link out

### PR DESCRIPTION
**Pull request by jgravois**
better to link out to _authoritative_ hub ready apps doc than maintain it here.

making sure no good stuff is lost entirely via https://github.com/ArcGIS/hub-ready-apps/issues/1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -32,6 +32,16 @@ e.g. for item "2f4c72dc471b4baab362c0437273bd51" put that on the tail end of the
 
 4. Enjoy the app!
 
+## [hub ready](https://github.com/Esri/hub-ready-apps/) checklist
+
+- [X] [Live Content](https://github.com/Esri/hub-ready-apps/blob/master/README.md#live-content)
```
Comment (jgravois) [inline README.md]: these `404`s will resolve soon. in the meantime: ~~Esri~~ > ArcGIS

**Comment by tomwayson** [comment]
LGTM, let's merge once those links will resolve

**Comment by MarkhamShofner** [comment]
Let's merge after Tom's ember-upgrade PR lands

**Comment by MarkhamShofner** [comment]
@jgravois do we expect the Esri/hub-ready-apps repo to be live by now?

**Comment by jgravois** [comment]
> do we expect the Esri/hub-ready-apps repo to be live by now?

it could be hours, days or even weeks until the admins 'flip the switch'. if you guys would rather hold off on merging this PR until they do, that's 👌  with me.

**Comment by jgravois** [comment]
💡

**Pull request merged**